sykmeldingalerts-dev.yaml

```diff
@@ -10,28 +10,28 @@ spec:
       channel: '#sykmelding-alerts-dev'
   alerts:
     - alert: syfosm-app-nede
-      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell|egenmeldt-sykmelding-backend"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"syfosmregler|syfosmregister|syfotekster|syfohelsenettproxy|syfosmpapirregler|syfosmmanuell-backend|syfosmmanuell|egenmeldt-sykmelding-backend|syfoservice-mq-producer"} == 0
```

Comment:
Dette er en asynkron app, den skal ikke være i alerten for de synkrone appene :)